pyanaconda/modules/storage/partitioning/custom_partitioning.py

```diff
@@ -37,7 +37,7 @@
 from pyanaconda.modules.storage.partitioning.schedule import get_candidate_disks, \
     schedule_partitions
 from pyanaconda.platform import platform
-from pyanaconda.storage.utils import get_available_disk_space, suggest_swap_size, get_pbkdf_args, \
+from pyanaconda.storage.utils import suggest_swap_size, get_pbkdf_args, \
```

Comment:
Please remove the `\` from this import, it should fit on one line now.

Reply:
Fixed.